pkg/symbol/elfutils/elfutils.go

```diff
@@ -153,3 +154,33 @@ func HasSymbols(filePath string) (bool, error) {
 	}
 	return false, nil
 }
+
+// ValidateFile returns true if the given object file is valid.
+func ValidateFile(path string) error {
```

Comment:
I have forgotten to update the comment :see_no_evil: I had a lot of back and forth on this. None of the errors is an unrecoverable error in this context. And the existence of the error indicates if it's valid or not. That's why I went for a simpler approach. Otherwise, error handling gets super messy.

Reply:
Sure thing! Either way is fine :D